.github/workflows/run-k8s-tests-on-garm.yaml

```diff
@@ -35,7 +35,15 @@ jobs:
           - devmapper
         k8s:
           - k3s
-    runs-on: garm-ubuntu-2004-smaller
+        instance:
+          garm-ubuntu-2004
+          garm-ubuntu-2004-smaller
+        include:
+          - instance: garm-ubuntu-2004
+            instance-type: normal
+          - instance: garm-ubuntu-2004-smaller
+            instance-type: small
+    runs-on: {{ matrix.instance }}
```

Comment:
are you sure we need the workflow runner to be big when the aks cluster uses a big node size? I would think that we could stick to a small runner even with bigger aks node instance, because the runner only executes remote commands.

Reply:
So, this is the "on-garm" case.  Here we're actually deploying a k8s cluster in a garm VM, as we need to setup things like the snapshotter being used.

In this case we're running the tests from inside the garm VM, which happens to also be the k8s cluster we setup, thus we need the correct size.  Does it make sense?

Reply:
now i follow - ok